Config.uk

```diff
@@ -2,6 +2,7 @@ menuconfig LIBCOMPILER_RT
     bool "compiler-rt - runtime support"
     select LIBPOSIX_SYSINFO
     select LIBUKMMAP
+    #XXX: libunwind dependency cannot be added due to circular dependency
```

Comment:
```suggestion
    # XXX: `select LIBUNWIND` cannot be added as it will cause circular dependency.
```
Moreover, there is a better solution as noted in the [Kconfig documentation](https://www.kernel.org/doc/html/latest/kbuild/kconfig-language.html#kconfig-recursive-dependency-limitations). Use `depends on`:

```
        depends on LIBUNWIND
```

Reply:
Since adding a dependency works using `depends on LIBUNWIND`, I will delete the comment from the file and update it with `depends on`.

Reply:
Perfect. Do that.